site/vite.config.js

```diff
@@ -1,10 +1,14 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
+import svgr from 'vite-plugin-svgr'
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [
     react(),
+    svgr({
+      include:  "**/icon-layers.svg?react",
```

Comment:
@eti7075 you'll need to add the refresh icon here to this include statement, otherwise the react-style import won't work.